### ISSUE TITLE
ConsoleApp throws Unable to load DLL WITDataStore32.dll

### ISSUE THREAD

**Issue by tomkuijsten** [tfsaggregator/tfsaggregator#103]
I tried testing a policy file when I encountered an error. This is my setup:
- Windows 2012 R2
- TFS 2015 Update 1
- TFSAggregator 2.0.138 release

I tried running this command:
`TFSAggregator2.ConsoleApp.exe run --policyFile=c:\install\TFSAggregator2.ServerPlugin.policies --teamProjectCollectionUrl=http://localhost/tfs/DefaultCollection --teamProjectName=SO --workItemId=22 --logLevel=diagnostic`

The error I got:

>C:\Program Files\TFS Aggregator\bin>TFSAggregator2.ConsoleApp.exe run --policyFi
le=c:\install\TFSAggregator2.ServerPlugin.policies --teamProjectCollectionUrl=ht
tp://localhost/tfs/DefaultCollection --teamProjectName=SO --workItemId=22 --logL
evel=diagnostic
Aggregator.ConsoleApp 2.0.138-master
Console tool to test TFS Aggregator polices
Build: 2.0.138, Configuration: Release
Copyright (c) TFS Aggregator Team 2015

>Executing run (Applies a policy file to specified work item(s)):

>[Information] 00.017 Aggregator.Core started, version: 2.0.138-master, build: 2.
0.138, configuration: Release
[Information] 00.080 Configuration loaded successfully from 'c:\install\TFSAggre
gator2.ServerPlugin.policies'
Error: Unable to load DLL 'Microsoft.WITDataStore32.dll': The specified module c
ould not be found. (Exception from HRESULT: 0x8007007E)

And my policy file (tried it with the default "TFSAggregator2.ServerPlugin.policies", same result):

```xml
<?xml version="1.0" encoding="utf-8"?>
<AggregatorConfiguration>
    
    <!-- Configure TFS Aggregator behavior. (Once, Optional)
     - **debug**: turns on debugging options (Optional, default: False)
    -->
    <runtime debug="false">
        <!--
        TBD
        -->
        <rateLimiting interval="00:00:01.0" changes="5" />
        <!--
        The level of logging. (Optional)
            * Critical
            * Error
            * Warning
            * Information or Normal - default value
            * Verbose
            * Diagnostic.
            -->
        <logging level="Normal" />
        <!--
        The language used to express the rules. (Optional)
        Valid values are:
            * CS,CSHARP,C# - default value
            * VB,VB.NET,VBNET
            * PS,POWERSHELL - *Experimental*!
        -->
        <script language="C#" />
        <!--
        autoImpersonate false (default) use the TFS Service account, true the user requesting. (Optional)
        -->
        <authentication autoImpersonate="false" />
    </runtime>
    
    <rule name="AutoOpen" appliesTo="Task">
		<![CDATA[
		if (self["System.State"] == "In progress" && self.HasParent && ((string)self.Parent["System.State"]) != "Committed")
		{
			self.Parent.TransitionToState("Committed", "Auto Activated");
		}
		]]>
	</rule>
    
<!-- 	<rule name="AutoClose" appliesTo="Task">
		<![CDATA[
		if (self["System.State"] == "Closed" && self.HasParent && ((string)self.Parent["System.State"]) != "Closed")
		{
			if (self.Parent.Children.Where(child => ((string)child.Parent["System.State"]) != "Removed").All(child => ((string)child.Parent["System.State"]) == "Closed")
			{
				self.Parent.TransitionToState("Closed", "Auto Closed");
			}
		}
		]]>
	</rule> -->
	
    <!-- Represent a set of aggregation rules that apply to a particular scope. (Repeatable)
    - **name**: The name of this policy. (Mandatory)
    -->
    <policy name="AutoOpenClose">
        <!-- Scope the policy to a list of collections. (Optional)
        - **collections**: The TFS Collection to which the policy applies. (List, Mandatory)
        '*' means any Collection
        -->
        <collectionScope collections="DefaultCollection" />
        <!-- Scope the policy to Team Projects using a particular Process Template.
        - **name**: Name of Process Template matching. (Optional)
            (Not working due to Microsoft not setting these values for on-premise installations)
        - **typeId**: Process Template GUID to match. (Optional)
        - **minVersion**: Minimum version for Process Template. (Optional)
        - **maxVersion**: Minimum version for Process Template. (Optional)
        -->
        <templateScope name="Scrum" />
        <!-- Microsoft Visual Studio Scrum 2013.4
        <templateScope typeId="6B724908-EF14-45CF-84F8-768B5384DA45" minVersion="3" maxVersion="3" />
        -->
        <!-- Scope the policy to listed Team Projects.
         - **projects**: List of Team Project names. (List, Mandatory)
        -->
        <projectScope projects="SO" />
        
        <!-- Reference to a previously declared rule.-->
		<ruleRef name="AutoOpen" />
    </policy>
    
    <!-- add more policies as needed -->
    
</AggregatorConfiguration>
```

**Comment by giuliov** [comment]
Sound like MSI failed to pick up `Microsoft.WITDataStore32.dll`. I will investigate in the next days, in the meantime are you able to copy that file from `C:\Program Files\Microsoft Team Foundation Server 14.0\Application Tier\web Services\bin` to `C:\Program Files\TFS Aggregator\bin`.
Let me know if this solves.

**Comment by tomkuijsten** [comment]
That was it, thanks! Adding it in the installer looks like the correct solution.

**Comment by giuliov** [comment]
Workaround is working but I will keep this open to track the fix.

**Comment by giuliov** [comment]
Fixed in 67bb9a8a4b26989f53f17d5fec792fa6b0c0bbf9
